modules/monitoring/library/Monitoring/DataView/DataView.php

```diff
@@ -442,7 +443,7 @@ public function applyFilter(Filter $filter)
      */
     public function validateFilterColumns(Filter $filter)
     {
-        if ($filter instanceof FilterMatch) {
+        if ($filter instanceof FilterMatch || $filter instanceof FilterEqual) {
```

Comment:
Just noticed, this should only check for `FilterExpression`. That's been wrong all these years already.